src/pages/edge/edge-network/tutorials/datastream-config-override-rules.md

```diff
@@ -0,0 +1,121 @@
+
+## Steps to add datastream config overrides for Lifecycle Edge events using rules
+
+Let's add **datastream ID override** or **datastream config overrides** for `Lifecycle Edge` events of type **application.launch** and **application.close** by following the next steps.
+
+### 1. Add a new rule for your mobile property
+
+Give your rule an easily recognizable name in your list of rules.
+In this example, the rule is named "Attach datastream config overrides Data to Lifecycle Edge events".
+
+### 2. Configure Lifecycle Edge Events
+
+Now we need to add two Lifecycle Edge events one of type **application.launch** and other of type **application.close**.
+
+#### Configure Lifecycle Edge event of type application.launch
+
+1. Under the **Events** section, select **Add**.
+2. From the **Extension** dropdown list, select **Adobe Experience Platform Edge Network**.
+3. From the **Event Type** dropdown list, select **AEP Request Event**.
+4. On the right pane, click the plus button **Add XDM Event Type** and set the condition to **equals** with value **application.launch**.
+5. Select **Keep Changes**.
+
+![Sample rule configuration application.launch event](../assets/tutorial/configOverrides/if-event-application-launch.png)
+
+#### Configure Lifecycle Edge event of type application.close
+
+1. Under the **Events** section, select **Add**.
+2. From the **Extension** dropdown list, select **Adobe Experience Platform Edge Network**.
+3. From the **Event Type** dropdown list, select **AEP Request Event**.
+4. On the right pane, click the plus button **Add XDM Event Type** and set the condition to **equals** with value **application.close**.
+5. Select **Keep Changes**.
+
+![Sample rule configuration application.close event](../assets/tutorial/configOverrides/if-event-application-close.png)
+
+Save the event configuration and return to the Rule Editor UI.
+
+### 3. Configure Attach Data Action
+
+Based on the use case you can **either** attach payload for the **datastream ID override** or for the **datastream config overrides**
+
+#### (Option 1) Configure the action to attach datastream ID override payload
+
+1. Under the **Actions** section, select **Add**.
+2. From the **Extension** dropdown list, select **Mobile Core**.
+3. From the **Action Type** dropdown list, select **Attach Data**.
+4. On the right pane, in the **JSON Payload** field, type the config overrides data that will be added to this event.
+
+    **Sample Payload for datastream ID override:**
+
+    ```json
+    {
+        "config": {
+            "datastreamIdOverride": "SampleDatastreamID"
+        }
+    }
+    ```
+
+    ![Action configuration - Datastream ID override payload ](../assets/tutorial/configOverrides/datastream-id-override-payload.png)
+
+5. Select **Keep Changes**.
+6. Return to the Rule Editor UI.
+
+**OR**
```

Comment:
No need for this or IMO - it's pretty implied that, just from having the sub-headings, that these are two potential options.